src/skip-wip-ci/skip-wip-ci.yml

```diff
@@ -0,0 +1,62 @@
+# Orb Version 0.1.0
+
+version: 2.1
+description: 'Use to skip CI when PR title contains "wip", "skip ci", "ci skip" or is draft'
+
+commands:
+  check-skippable-pr:
+    steps:
+      - run:
+          shell: /bin/bash
+          name: Check Skippable PR
+          command: |
+            required_env_vars=(
+              "BUNDLE_GITHUB__COM"
+              "CIRCLE_PROJECT_USERNAME"
+              "CIRCLE_PR_REPONAME"
+              "CIRCLE_PR_NUMBER"
+            )
+
+            for required_env_var in ${required_env_vars[@]}; do
```

Comment:
There's nothing to change here, but we could definitely abstract out stuff like this (the ability to check for env vars) to a base orb or something. cc @dblandin this came up with something else we were looking at too... I forget now what that was.

Reply:
It was for the `AWS_*` environment variables check in the remote-docker orb: https://github.com/artsy/orbs/blob/70d3e1c42b5b981e009ed49a7199496909776106/src/remote-docker/remote-docker.yml#L23

Reply:
yeah that's a good point, maybe a base orb that handles PR stuff? My only hesitation is that other orbs that use the base orb may have different requirements.